comid-triples.cddl

```diff
@@ -0,0 +1,142 @@
+concise-mid-tag = {
+  ; TODO(tho) language
+  ; TODO(tho) tag-id
+  ; TODO(tho) module-name
+  ; TODO(tho) entity
+  ; TODO(tho) linked-tags
+  comid.claims => claims-map
+  ; TODO(tho) extension
+}
+
+claims-map = non-empty<{
+  comid.reference-claims => one-or-more<reference-claim-record>
+  comid.endorsed-claims => one-or-more<endorsed-claim-record>
+  comid.identity-claims => one-or-more<identity-claim-record>
+  * $$claims-map-extension
+}>
+
+; REFERENCE-VALUE triple
+; "target environment $TE" "has reference measurements" "$RV"
+reference-claim-record = {
+  target-environment: environment-map 
+  reference-values: values-map
+}
+
+; ENDORSED-VALUE triple
+; "environment $E" "has endorsed measurements" "$EV"
+endorsed-claim-record = {
+  environment: environment-map 
+  endorsed-values: values-map
+}
+
+; DEVICE-IDENTITY triple
+; (Should it be AE-IDENTITY instead?)
+; "attesting environment $AE" "signs Evidence with key" "$K"
+identity-claim-record = {
+  attesting-environment: environment-map 
+  identity: COSE_Key
```

Comment:
I think, we should allow the flexibility of  one-or-more<COSE_Key> here!
A Class or an Instance may have more than one key associated to it!

Reply:
Plus there could be may DICE Cert Provisions which we want to address using single container!

Reply:
In 5b, at the moment, there is no multiplicity.  If we want to have one-or-more we need to change the IM first.

Reply:
I read Yogesh's point to mean there could be backup keys that would be treated equally by the verifier (essentially clone keys). I didn't interpret it to mean that each instance of COSE_Key would be different key in the cert chain.